README.md

```diff
@@ -4,7 +4,6 @@
 
 [![Build Status](https://travis-ci.org/mohammad-fouladgar/eloquent-builder.svg?branch=master)](https://travis-ci.org/mohammad-fouladgar/eloquent-builder)
 [![Coverage Status](https://coveralls.io/repos/github/mohammad-fouladgar/eloquent-builder/badge.svg?branch=develop)](https://coveralls.io/github/mohammad-fouladgar/eloquent-builder?branch=develop)
-[![Quality Score](https://img.shields.io/scrutinizer/g/mohammad-fouladgar/eloquent-builder.svg?style=flat-square)](https://scrutinizer-ci.com/g/mohammad-fouladgar/eloquent-builder)
 [![StyleCI](https://github.styleci.io/repos/144369188/shield?branch=develop)](https://github.styleci.io/repos/144369188)
```

Comment:
I think we no longer need this badge, right? We can use GitHub action for this.

Reply:
Yes, that’s why I removed the badge.

Reply:
I mean the StyleCI badge.

Reply:
@mohammad-fouladgar Hah! Yes, you're right! So, please as you're bumping a version up, remove this one as well.